data/mods/ssb/moves.js

```diff
@@ -2295,6 +2295,43 @@ let BattleMovedex = {
 		type: "Fairy",
 		zMoveEffect: 'clearnegativeboosts',
 	},
+	// Mad Monty ¾°
+	"llamacide": {
+		accuracy: 100,
+		basePower: 80,
+		category: "Physical",
+		desc: "This attack has a 40% chance to lower foe's Defense, and a 10% chance to Freeze it.",
+		shortDesc: "40% chance to lower foe's Defense, 10% to Freeze",
+		id: "llamacide",
+		name: "Llamacide",
+		pp: 10,
+		flags: {},
+		secondary: {
```

Comment:
Just use `secondaries: [],`, which allows you to place multiple objects containing individual secondary effects into it.

https://github.com/Zarel/Pokemon-Showdown/blob/master/sim/dex-data.ts#L734